inspec/a2-api-integration/controls/iam/iam_apis.rb

```diff
@@ -1,416 +1,29 @@
-      expect(resp.http_status).to eq 200
-
-      resp = automate_api_request("/apis/iam/v2/roles")
-      expect(resp.http_status).to eq 200
-      expect(resp.parsed_response_body[:roles].length).to eq init_role_count
-    end
-
-    it "LIST roles responds properly" do
-      resp = automate_api_request("/apis/iam/v2/roles")
-      expect(resp.http_status).to eq 200
-    end
-
-    it "GET role responds properly to happy path inputs" do
-      resp = automate_api_request("/apis/iam/v2/roles/#{CUSTOM_ROLE_ID}")
-      expect(resp.http_status).to eq 200
-    end
-
-    it "UPDATE role responds properly to happy path inputs" do
-      resp = automate_api_request("/apis/iam/v2/roles/#{CUSTOM_ROLE_ID}",
-        http_method: 'PUT',
-        request_body: {
-          name: "inspec test role updated",
-          actions: ["brand:new:action"]
-        }.to_json
-      )
-      expect(resp.http_status).to eq 200
-      expect(resp.parsed_response_body[:role][:name]).to eq "inspec test role updated"
-      expect(resp.parsed_response_body[:role][:actions]).to eq ["brand:new:action"]
-    end
-  end
-
-  describe "v2 project API" do
-    custom_project_id = "inspec-custom-project-api-test-#{TIMESTAMP}"
-    before(:all) do
-       resp = automate_api_request("/apis/iam/v2/projects",
-        http_method: 'POST',
-        request_body: {
-          id: custom_project_id,
-          name: "display name !#$#"
-        }.to_json
-      )
-      expect(resp.http_status).to eq 200
-    end
-
-    after(:all) do
-      resp = automate_api_request("/apis/iam/v2/projects/#{custom_project_id}", http_method: 'DELETE')
-      expect(resp.http_status).to eq 200
-    end
-
-    it "LIST projects responds properly" do
-      resp = automate_api_request("/apis/iam/v2/projects")
-      expect(resp.http_status).to eq 200
-    end
-
-    it "CREATE and DELETE project properly respond to happy path inputs" do
-      resp = automate_api_request("/apis/iam/v2/projects")
-      expect(resp.http_status).to eq 200
-      init_project_count = resp.parsed_response_body[:projects].length
-
-      id = "inspec-project-#{Time.now.utc.to_i}"
-      resp = automate_api_request("/apis/iam/v2/projects",
-        http_method: 'POST',
-        request_body: {
-          id: id,
-          name: "display name !#$#"
-        }.to_json
-      )
-      expect(resp.http_status).to eq 200
-      expect(resp.parsed_response_body[:project][:id]).to eq id
-      expect(resp.parsed_response_body[:project][:name]).to eq 'display name !#$#'
-
-      resp = automate_api_request("/apis/iam/v2/projects")
-      expect(resp.http_status).to eq 200
-      expect(resp.parsed_response_body[:projects].length).to eq init_project_count + 1
-      expect(resp.parsed_response_body[:projects].map { |p| p[:id] }).to include id
-
-      resp = automate_api_request("/apis/iam/v2/projects/#{id}", http_method: 'DELETE')
-      expect(resp.http_status).to eq 200
+title 'iam REST API integration tests'
```

Comment:
this file now tests all IAM APIs: tokens, users, teams, policies, projects + rules, roles